python/ql/src/Exceptions/EmptyExcept.ql

```diff
@@ -64,32 +64,29 @@ predicate subscript(Stmt s) {
     s.(Delete).getATarget() instanceof Subscript
 }
 
-predicate encode_decode(Expr ex, ClassObject type) {
+predicate encode_decode(Expr ex, ClassValue type) {
     exists(string name |
         ex.(Call).getFunc().(Attribute).getName() = name |
```

Comment:
:thinking: It's sort of odd that we only use `ex` when cast to `Call`. It should be easier just to change the predicate signature to expect a `Call` in the first place.

Reply:
Done!